framework/src/utils/MooseUtils.C

```diff
@@ -355,8 +355,8 @@ indentMessage(const std::string & prefix, std::string & message, const char* col
   // The colored prefix
   std::string indent = color + prefix + ": " + COLOR_DEFAULT;
 
-  // Indent all lines after the first
-  pcrecpp::RE re("\n(?!\\Z)");
+  // Indent all lines
+  pcrecpp::RE re("\n");
```

Comment:
I actually try to move the code in Console.C to here but it does not work. There is something going on i am not aware of. That is why I want Andrew the original developer to take a look at.